zenodb_test.go

```diff
@@ -365,12 +365,20 @@ view_a:
 		}
 		return true, nil
 	})
-	testAggregateQuery(t, db, now, epoch, resolution, asOf, until, scalingFactor, modifyTable)
+
+	testAggregateQuery(t, db, now, epoch, resolution, asOf, until, scalingFactor, false, modifyTable)
+	testAggregateQuery(t, db, now, epoch, resolution, asOf, until, scalingFactor, true, modifyTable)
 }
 
-func testAggregateQuery(t *testing.T, db *DB, now time.Time, epoch time.Time, resolution time.Duration, asOf time.Time, until time.Time, scalingFactor int, modifyTable func(string, func(*table))) {
+func testAggregateQuery(t *testing.T, db *DB, now time.Time, epoch time.Time, resolution time.Duration, asOf time.Time, until time.Time, scalingFactor int, stride bool, modifyTable func(string, func(*table))) {
 	log.Debugf("AsOf: %v  Until: %v", asOf, until)
 
+	period := resolution * time.Duration(scalingFactor)
+	periodString := fmt.Sprintf("period(%v)", period)
+	if stride {
+		periodString = fmt.Sprintf("%v, stride(%v)", periodString, period)
```

Comment:
This is kind of a silly test, because setting the stride and period to the same value yields the same thing as just omitting stride. Still, it makes sure that including stride doesn't break anything.